src/libs/OptionsListUtils.ts

```diff
@@ -1451,13 +1467,16 @@ function getValidOptions(
         shouldSeparateSelfDMChat = false,
         shouldSeparateWorkspaceChat = false,
         excludeHiddenReports = false,
+        canShowManagerMcTest = false,
         ...config
     }: GetOptionsConfig = {},
 ): Options {
     // Gather shared configs:
     const loginsToExclude: Record<string, boolean> = {
         [CONST.EMAIL.NOTIFICATIONS]: true,
         ...excludeLogins,
+        // Exclude Manager McTest if user submitted expense or scanned receipt and when selection is made from Create or Submit flow
+        [CONST.EMAIL.MANAGER_MCTEST]: !(Permissions.canUseManagerMcTest(config.betas) && !getIsUserSubmittedExpenseOrScannedReceipt() && canShowManagerMcTest),
```

Comment:
NAB: this block is harder to compute in your head with the `!`, can we do the inverse?

i.e. `Permissions.canUseManagerMcTest(config.betas) || getIsUserSubmittedExpenseOrScannedReceipt()`

Reply:
Actually, I think it's fine for this case since I missed the third